router/router.go

```diff
@@ -236,6 +236,10 @@ func (r *Router) serveHTTPS(server *http.Server, errChan chan error) error {
 		ClientAuth:   r.config.ClientCertificateValidation,
 	}
 
+	if r.config.EnableHTTP2 {
+		tlsConfig.NextProtos = []string{"h2"}
```

Comment:
❓ Can you explain what this does?

Reply:
Adding `h2` to the TLS config's list of available protocols lets the tlsListener try to negotiate HTTP/2 via ALPN. From the `Serve` function documentation:

> HTTP/2 support is only enabled if the Listener returns *tls.Conn connections and they were configured with "h2" in the TLS Config.NextProtos. - https://golang.org/pkg/net/http/#Serve